### DISCUSSION TITLE
Upgrading from 1.1.5 to 1.1.6

### DISCUSSION THREAD

**Discussion by Yuvix25** [Chia-Network/chia-blockchain#5674] [Support]
Hi, I have seen that version 1.1.6 of the blockchain is now available.
Because I have a few issues with 1.1.5 (mostly syncing related), I would like to upgrade.
My questions are: 
Do I need to stop the plotting? (I am using swar-chia-plot-manager)
Will I have to resync all the blockchain or it will be saved?
Will all my completed plots be saved?

Thanks in advance.

**Comment by rthorntn** [comment]
Only one I can answer is you wont lose your plots.

**Comment by mariano54** [comment]
Will I have to resync all the blockchain or it will be saved?
saved

**Comment by Daniel-1276** [comment]
You need to stop plotting for sure, as chia.exe / binaries need to be exchanged during update. I did 2 upgrades already and already synced blockchain data was preserved, but of course, like on any stop of the GUI you need to resync. Resync should only take a couple of minutes, Completed plots are your's and using your harmonic keywords you can always reuse them even on another PC.

**Reply by Yuvix25** [reply]
Ok thanks! This is what I thought would happen just wanted to be sure

**Comment by BrandtH22** [comment]
@Yuvix25 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
